Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointerVisual.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
+    [ExecuteAlways]
```

Comment:
This didnt' have execute in edit mode, why did you add ExecuteAlways here?

Reply:
To ensure OnEnable / CheckInitialization is called in edit mode. This is a substitute for OnValidate.